java/fury-core/src/main/java/org/apache/fury/memory/MemoryBuffer.java

```diff
@@ -1316,24 +1316,15 @@ public int readPositiveVarInt() {
   }
 
   private int readPositiveVarIntSlow() {
-    int b = readByte();
-    int result = b & 0x7F;
-    if ((b & 0x80) != 0) {
+    int shift = 0;
```

Comment:
Could we benchmark the performance of those two implementation using JMH? The previous one are loop unrolled for performance

Reply:
The implementation in this class are kind of messy, we manually inline code and unrolled some loop for performance

Reply:
Yes, could you add some comments in this pr? I think we'd better add comments to similar methods

Reply:
Updated.

BTW, do we need to add the benchmark code (sample code above) that uses loop and does not use loop in the `MemoryBuffer#readPositiveVarIntSlow` method to the benchmark directory?

Reply:
I don't think it's necessary, only rolling code will be used in the release